router/mount.go

```diff
@@ -11,18 +13,43 @@ var HttpMethods = []string{"OPTIONS", "GET", "HEAD", "POST", "PUT", "DELETE", "P
 // ServerlessRouterMount represents a single mount point on the API
 // Such as '/path', the HTTP method, and the function to resolve it
 type ServerlessRouterMount struct {
-	Name     		string
-	Function 		*AWSServerlessFunction
-	Handler  		http.HandlerFunc
-	Path     		string
-	Method   		string
+	Name             string
+	Function         *AWSServerlessFunction
+	Handler          http.HandlerFunc
+	Path             string
+	Method           string
+	BinaryMediaTypes []string
 
 	// authorization settings
 	AuthType       string
 	AuthFunction   *AWSServerlessFunction
 	IntegrationArn *LambdaFunctionArn
 }
 
+// Returns the wrapped handler to encode the body as base64 when binary
+// media types contains Content-Type
+func (m *ServerlessRouterMount) WrappedHandler() http.HandlerFunc {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		contentType := req.Header.Get("Content-Type")
+		binaryContent := false
+
+		for _, value := range m.BinaryMediaTypes {
+			if value == contentType {
```

Comment:
Not sure you can make a naive comparison. My request Content-Type header value may be `multipart/form-data; boundary=something` and it would not match multipart/form-data in your map.

Reply:
I used `mime.ParseMediaType` to strip parameters.